src/promptflow/promptflow/_constants.py

```diff
@@ -68,6 +68,7 @@ class AvailableIDE:
 # trace related
 OTEL_RESOURCE_SERVICE_NAME = "promptflow"
 DEFAULT_SPAN_TYPE = "default"
+RUNNING_LINE_RUN_STATUS = "Running"
```

Comment:
Another minor change, the status `Ok` is strange, can make it to `OK`? May do this in next pr tho.
![image](https://github.com/microsoft/promptflow/assets/2572521/dabccd67-d48d-48eb-87bd-9730fcbb8ab9)

Reply:
And why is it `OK` instead of `Completed` which is more aligned with our run status in the first place?

Reply:
ah, this is something that open telemetry defines: <https://opentelemetry.io/docs/specs/otel/trace/api/#set-status>

![image](https://github.com/microsoft/promptflow/assets/38847871/4618e062-1ce2-4c94-9ad9-0dd8ce97e9ee)

i don't like it, but, that is the standard...